tests/distgit_test.py

```diff
@@ -0,0 +1,777 @@
+#!/usr/bin/env
```

Comment:
Is this file just bump from `doozerlib/distgit_test.py` any reason not bumping the rest unittest files as well, like rpmcfg_test, runtime_test etc... ?

Reply:
yes, still a @wip, just started doing one by one (since some of the other *_test.py files contain failing tests)